tee-worker/ts-tests/integration-tests/common/utils/assertion.ts

```diff
@@ -146,11 +146,35 @@ export async function assertVc(context: IntegrationTestContext, subject: CorePri
 
     // step 7
     // check runtime version is present
-    assert.deepEqual(
-        vcPayloadJson.issuer.runtimeVersion,
-        { parachain: 9195, sidechain: 109 },
-        'Check VC runtime version: it should equal the current defined versions'
+    const specPaths = ['../../../runtime/litentry/src/lib.rs', '../../app-libs/sgx-runtime/src/lib.rs'].map(
```

Comment:
to confirm: are these the right sources?

Reply:
Our CI uses rococo I believe